scripts/prepareForFirstWeekLaunch.ts

```diff
@@ -1,24 +1,24 @@
-import {
-  BIG_NUMBER_1E18,
-  MAX_UINT256,
-  asyncForEach,
-  getCurrentBlockTimestamp,
-  increaseTimestamp,
-} from "../test/testUtils"
-import {
-  GaugeController,
-  GaugeHelperContract,
-  GenericERC20,
-  Minter,
-  SDL,
-  VotingEscrow,
-} from "../build/typechain/"
-
 import chai from "chai"
 import { ethers } from "hardhat"
 import { solidity } from "ethereum-waffle"
 
 const { expect } = chai
+import {
+  GaugeController,
+  SDL,
+  GaugeHelperContract,
+  VotingEscrow,
+  Minter,
+  LiquidityGaugeV5,
+  GenericERC20
+} from "../build/typechain"
+import {
+  BIG_NUMBER_1E18,
+  getCurrentBlockTimestamp,
+  MAX_UINT256,
+  setTimestamp,
+  asyncForEach,
+} from "../test/testUtils"
 
 chai.use(solidity)
 
```

Comment:
Could you add a tldr of what this script does at the top? Like "unpause SDL, set gauge weights, create and distribute dummy reward erc20's to gauges..."